tests/src/integration/runtimetests/manifest.yml

```diff
@@ -0,0 +1,62 @@
+package:
```

Comment:
@lzbj I do not see tests fro Python2 and Python3, which are both supported currently by OpenWhisk.  i.e., 
"image": "openwhisk/python2action:latest",
"deprecated": false,
"requireMain": false,
"default": true,
"attached": false,
"kind": "python:2"
}, {
"image": "openwhisk/python3action:latest",
"deprecated": false,
"requireMain": false,
"default": false,
"attached": false,
"kind": "python:3"
}],

Reply:
We should likely test for python:2, python:3, nodejs:5 and nodejs:6 (and what happens if we use python2.7 (point versions)?

Reply:
ok, add test